recipe/meta.yaml

```diff
@@ -1,17 +1,19 @@
 {% set version = "6.4" %}
+{% set patch = "20240210" %}
+{% set pkg_version = version + ("_" + patch if patch else "") %}
```

Comment:
I also have a question about the logic that updates component versions, will it be able to update with the patch version? @ocefpaf @h-vetinari @jakirkham @jmarshall Do you know anything about how it works on conda-forge side? Perhaps the current logic will not be able to handle the modification of the version if it is such

Reply:
We should try to use release for the auto-updating bot to work properly. I don't think it will work with a git tag.

Reply:
Well, they say about https://github.com/ThomasDickey/ncurses-snapshots.git, there are only tags here ☹️

Reply:
You can still use the URL form, like:

```yaml
url: https://github.com/ThomasDickey/ncurses-snapshots/archive/refs/tags/{{ version }}.tar.gz
sha256: ....

Reply:
It's better to do any computation on the tag after it has been been picked up by the bot; otherwise, if you hardcode the patch value here, the bot will never find an update by itself. You can check the rstudio feedstock for an example of what I'm talking about.

Reply:
I did update it to my best knowledge based on @h-vetinari  and @ocefpaf  recommendations. I was trying to setup the autotick bot locally to be able to test if it was working, but I failed with understanding how (and if) it can be used to test if the recipe supports automatic updates.